### DISCUSSION TITLE
Present at FOSDEM?

### DISCUSSION THREAD

**Discussion by Ainali** [publiccodeyml/publiccode.yml#128] [Ideas]
I think a presentation about publiccode.yml would be excellent in the Public Code devroom at FOSDEM. I would really much look forward to seeing a submission from the community. 

The call for proposals is here: https://blog.publiccode.net/news/2021/12/01/fosdem-2022-public-code-devroom-call-for-proposal.html

**Comment by ruphy** [comment]
@Ainali I think that it absolutely makes sense!

**Comment by Ainali** [comment]
Thanks for the submission, it has been approved!
